bundles/io/org.eclipse.smarthome.io.transport.serial.ctrl/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.smarthome.io.transport.serial.ctrl</name>
```

Comment:
I do not think we need this separate project/bundle. After all, the classes are in the `org.eclipse.smarthome.io.transport.serial.internal` package, so they should best be part of the `org.eclipse.smarthome.io.transport.serial` bundle. It is good enough if API and Impl are separated by package, we do not require separate bundles (and we do not do this at other places in ESH).